examples/cassandra/README.md

```diff
@@ -57,103 +71,51 @@ command line tool somewhere in your path.  Please see the
-  volumes:
-    - name: data
-      emptyDir: {}
-```
+# delete the replication controller
+kubectl delete rc cassandra
 
-[Download example](cassandra.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE cassandra.yaml -->
+# then, create a daemonset to place a cassandra node on each kubernetes node
+kubectl create -f examples/cassandra/cassandra-daemonset.yaml
 
-There are a few things to note in this description.  First is that we are
-running the [```gcr.io/google-samples/cassandra:v8```](image/Dockerfile)
-image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
+# resource cleanup
+kubectl delete service -l app=cassandra
+kubectl delete daemonset cassandra
+```
 
-This is a standard Cassandra installation on top of Debian.  However it also
-adds a custom
-[`SeedProvider`](https://svn.apache.org/repos/asf/cassandra/trunk/src/java/org/apache/cassandra/locator/SeedProvider.java) to Cassandra.  In
-Cassandra, a ```SeedProvider``` bootstraps the gossip protocol that Cassandra
-uses to find other nodes.
-The [`KubernetesSeedProvider`](java/src/io/k8s/cassandra/KubernetesSeedProvider.java)
-discovers the Kubernetes API Server using the built in Kubernetes
-discovery service, and then uses the Kubernetes API to find new nodes (more on
-this later). See the [image](image/) directory of this example for specifics on
-how the container image was built and what it contains.
 
-You may also note that we are setting some Cassandra parameters (`MAX_HEAP_SIZE`
-and `HEAP_NEWSIZE`) and adding information about the
-[namespace](../../docs/user-guide/namespaces.md).
-We also tell Kubernetes that the container exposes
-both the `CQL` and `Thrift` API ports.  Finally, we tell the cluster
-manager that we need 0.1 cpu (0.1 core).
 
-In theory, we could create a single Cassandra pod right now, but since
-`KubernetesSeedProvider` needs to learn what nodes are in the Cassandra
-deployment we need to create a service first.
+## Step 1: Create a Cassandra Service
 
-### Cassandra Service
+A Kubernetes _[Service](../../docs/user-guide/services.md)_ describes a set of
+[_Pods_](../../docs/user-guide/pods.md) that perform the same task. In
+Kubernetes, the atomic unit of an application is a Pod: one or more containers
+that _must_ be scheduled onto the same host.
 
-In Kubernetes, a _[Service](../../docs/user-guide/services.md)_ describes a set
-of Pods that perform the same task.  For example, the set of Pods in a Cassandra
-cluster can be a Kubernetes Service, or even just the single Pod we created
-above.  An important use for a Service is to create a load balancer which
-distributes traffic across members of the set of Pods.  But a _Service_ can also
-be used as a standing query which makes a dynamically changing set of Pods (or
-the single Pod we've already created) available via the Kubernetes API.  This is
-the way that we use initially use Services with Cassandra.
+An important use for a Service is to create a load balancer which
```

Comment:
I think in this case Brendan did mean something a bit different, which that the seedprovider uses the endpoint API. So in that case it's not doing load balancing.